From a8efa2f6115dc3f7d5acaf90342d61df9fedea1b Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 20 Sep 2016 15:47:48 +0800
Subject: [PATCH] ARM64: dts: rk3399-evb: disable dmc

Since ddr initalize codes have some update, so ddr dmc driver
needs to update.

Disable the dmc currently for evb to boot success.

Change-Id: Ica259a81f5412da31d41b68c841dfda7d8c2e3b2
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
index 9f7f3b526873..9034f209e4d1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
@@ -310,11 +310,11 @@
 };
 
 &dfi {
-	status = "okay";
+//	status = "okay";
 };
 
 &dmc {
-	status = "okay";
+//	status = "okay";
 	center-supply = <&vdd_center>;
 	upthreshold = <60>;
 	downdifferential = <40>;
-- 
2.35.3

